src/Card/Card.stories.js

```diff
@@ -28,39 +28,24 @@ export default {
   },
   argTypes: {
     element: {
-      control: {
```

Comment:
From what console says, control.options is deprecated and will be removed in version `7`.

Reply:
Good to know...